bom/pom.xml

```diff
@@ -114,6 +116,11 @@
         <version>${dropwizard-metrics.version}</version>
       </dependency>
       <dependency>
+        <groupId>io.dropwizard</groupId>
+        <artifactId>dropwizard-metrics-graphite</artifactId>
+        <version>${dropwizard-metrics-graphite.version}</version>
```

Comment:
I changed now to the aligned version of graphite - so all your proposed changes are made.